src/index.js

```diff
@@ -221,6 +221,11 @@ class Dropzone extends React.Component {
         rejectedFiles.push(...acceptedFiles.splice(0))
       }
 
+      // Update `acceptedFiles` and `rejectedFiles` state
+      // This will make children render functions receive the appropriate
+      // values
+      this.setState({ acceptedFiles, rejectedFiles })
```

Comment:
Actually, this still does not have any guarantee that the cb fn will be called after the state is set (`.setState()` is async).

What you want is to have the onDrop* cb fns called in the `.setState({...}, () => /* call the onDrop* fns here */)` cb fn 😄

Also, add a test for this.